### PR TITLE
Starting another bunch

### PR DESCRIPTION
**Pull request by dougaitken**
I've going through the list and deleting companies who aren't looking for employees, only contributors (like Android Authority for blog articles) or similar.

Just ship this one when you're ready so I can get an employee of one of the companies to contribute.

### CODE REVIEW

File: company-profiles/atomic-squirrel.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Atomic Squirrel
+
+## Company blurb
+
+Atomic Squirrel specializes in helping startups get off the ground.
+
+## Company size
+
+Team of 2.
```
Comment (dougaitken) [inline company-profiles/atomic-squirrel.md]: @brikis98 - when this is merged, can you edit this profile to fill out the information please? Thanks

Comment (brikis98) [inline company-profiles/atomic-squirrel.md]: Sure, thanks!

**Comment by jessicard** [comment]
That totally makes sense. Awesome, thank you as always @dougaitken :pray:

File: company-profiles/atomic-squirrel.md (hunk 1 above)
Comment (jessicard) [inline company-profiles/atomic-squirrel.md]: @brikis98 Merged!

**Pull request merged**